Tasks/Gradle/gradletask.ts

```diff
@@ -64,7 +66,7 @@ if (isCodeCoverageOpted && inputTasks.indexOf('clean') == -1) {
     gb.arg('clean'); //if user opts for code coverage, we append clean functionality to make sure any uninstrumented class files are removed
 }
 
-gb.argString(tl.getInput('options', false));
```

Comment:
`.arg` appends a single arg.
`.line` breaks on unquoted spaces and appends as multiple args.